### PR TITLE
Publish near-vm-* to crates.io

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: runtime/near-vm-runner/Cargo.toml [hunk 1]
```diff
@@ -3,16 +3,24 @@ name = "near-vm-runner"
...
+categories = ["wasm"]
+repository = "https://github.com/nearprotocol/nearcore"
+homepage = "https://github.com/nearprotocol/nearcore"
+description = """
+This crate implements the specification of the interface that Near blockchain exposes to the smart contracts.
+"""
 
 [dependencies]
-cached = { git = "https://github.com/nearprotocol/cached", rev = "7e472eddef68607e344d5a106a0e6705d92e55be" }
+near-cached = "0.8.0"
```
Comment (mikhailOK) [inline runtime/near-vm-runner/Cargo.toml]: can use regular cached 0.9.0

Comment (nearmax) [reply runtime/near-vm-runner/Cargo.toml]: Did they fix it?

Comment (mikhailOK) [reply runtime/near-vm-runner/Cargo.toml]: PR from near branch got merged

Comment (nearmax) [reply runtime/near-vm-runner/Cargo.toml]: Fixed.

**Pull request merged**